website/docs/index.html.markdown

```diff
@@ -36,25 +32,10 @@ The AWS provider offers a flexible means of providing credentials for
 authentication. The following methods are supported, in this order, and
 explained below:
 
-- Static credentials
```

Comment:
Whether we recommend their usage or not, this authentication method list is ordered. We need to keep static credentials at the top as they take priority over all others.

You mention in your pull request comment that the goal is to encourage other methods so I believe rather than changing what is here we should just follow through by adding those recommendations in that documentation section (e.g. using `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables instead).

Reply:
Static credential section has been moved to top as suggested. Fixed in 945b612